stats2480.osc.edu/apps/bc_osc_rstudio_server/form.yml

```diff
@@ -0,0 +1,13 @@
+---
+cluster: "owens"
+form:
+  - version
+  - bc_account
+  - bc_num_hours
+  - node_type
+  - num_cores
+attributes:
+  num_cores: 1
+  bc_account: "PAS1579"
+  node_type: "any"
+  version: "gnu/7.3.0 mkl/2018.0.3 R/3.6.0 rstudio/1.1.380_server"
```

Comment:
Assuming that they want the newest version of R that OSC has available until we hear otherwise.